### DISCUSSION TITLE
v2.2.7 release date

### DISCUSSION THREAD

**Discussion by RKS-labs** [deepmodeling/deepmd-kit#2949] [General]
I noticed that there have been two bug fixes for the `se_a` compression since v2.2.6. I know the bugs only affect edge cases, but I prefer to switch to a relatively stable version for long-term reproducibility while taking advantage of the `se_a_ebd_v2` compression.

**Comment by njzjz** [comment]
It's okay to me. cc @wanghan-iapcm

**Reply by wanghan-iapcm** [reply]
I agree.

**Comment by njzjz** [comment]
We've released v2.2.7.
